tests/test_modules/test_model_performance/test_confusion_matrix.py

```diff
@@ -56,3 +57,17 @@ def test_confusion_matrix(tiny_text_config):
     )
     [json_output_not_normalized] = mod_not_normalized.compute_on_dataset_split()
     assert json_output_not_normalized.confusion_matrix.sum() == dm.num_rows
+    assert [json_output_not_normalized] == [
+        ConfusionMatrixResponse(confusion_matrix=json_output_not_normalized, normalized=False)
+    ]
+
+    # When normalized, we get the predictions.
+    mod_not_normalized = ConfusionMatrixModule(
```

Comment:
I guess you didn't meant to call this "not normalized".
```suggestion
    mod_normalized = ConfusionMatrixModule(
```

Reply:
that was test script to check the normalized attribute value.